Philips.CodeAnalysis.Test/Maintainability/Maintainability/AvoidExcludeFromCodeCoverageAnalyzerTest.cs

```diff
@@ -0,0 +1,78 @@
+﻿// © 2023 Koninklijke Philips N.V. See License.md in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Philips.CodeAnalysis.MaintainabilityAnalyzers.Maintainability;
+using Philips.CodeAnalysis.Test.Helpers;
+using Philips.CodeAnalysis.Test.Verifiers;
+
+namespace Philips.CodeAnalysis.Test.Maintainability.Maintainability
+{
+	[TestClass]
+	public class AvoidExcludeFromCodeCoverageAnalyzerTest : DiagnosticVerifier
+	{
+		[DataTestMethod]
```

Comment:
Suggest a DataRow with multiple attributes. (There is a singlebaseanalyzer based on an AttributeList, so curious how those interplay.)

Reply:
I could base this Analyzer also on AttributeList if that aligns more